file-formats/clas/clas.json

```diff
@@ -1,78 +1,80 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "http://github.com/SAP/abap-file-formats/file-formats/clas/clas.json",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/clas/clas.json",
+  "title": "Interface Properties",
+  "description": "Interface properties",
```

Comment:
Should be `Class Properties`, but it is fixed by the changes to the ABAP Doc in the interface, see also other comment